fullrt/dht.go

```diff
@@ -852,24 +852,28 @@ func (dht *FullRT) execOnMany(ctx context.Context, fn func(context.Context, peer
 	}
 
 	var numFail, numSuccess int
-	t := time.NewTimer(0)
-	if !t.Stop() {
-		<-t.C
-	}
-	defer t.Stop()
+	var t *time.Timer
+	var toChan <-chan time.Time
```

Comment:
nit: I assume `toChan` is short for `timeoutChan` I'd just use the full thing to make it clearer since `to` is also the word to.

Reply:
done